### ISSUE TITLE
Remaining fragments after cancel

### ISSUE THREAD

**Issue by kingfisher77** [e-alfred/flowupload#176]
Hi, we are wondering what happens to the fragments of files which remains on the server when we cancel an upload. Are they meant to stay? Should we maintain the deletion with a separate process?

### Steps to reproduce
1. Upload big file (for example 5G)
2. Cancel upload after a while
3. All the 1M files remain on the server

### Expected behaviour
The fragments should be deleted

### Actual behaviour
The fragments remain.

### Server configuration

**Operating system**:
CentOS 8

**Web server:**
nginx

**Database:**
PosgreSQL

**PHP version:**
7.4

**Nextcloud version:** (see Nextcloud admin page)
latest

**Comment by JonathanTreffler** [comment]
Yes, currently chunks only get deleted if you finish the upload. This is necessary for you to be able to continue uploads, even if the tab got closed.

It would be great if they would get deleted after a month or so, but we have not implemented that (yet ?).
It would not be very easy to implement, but not impossible.
BTW: You can safely delete chunks of old uploads.
